source/include/core_mqtt.h

```diff
@@ -428,9 +438,10 @@ MQTTStatus_t MQTT_Init( MQTTContext_t * pContext,
  * publishes.
  * @param[in] incomingPublishCount Maximum number of records which can be kept in the memory
  * pointed to by @p pIncomingPublishRecords.
- *
- * @return #MQTTBadParameter if invalid parameters are passed;
- * #MQTTSuccess otherwise.
+ * @param[in] pAckPropsBuf Pointer to memory which will be used to store properties of outgoing publish-ACKS.
+ * @param[in] ackPropsBufLength Length of the buffer pointed to by @p pBuffer.
+ * @return #MQTTBadParameter if invalid parameters are passed;<br>
```

Comment:
Same here - why the line breaks?

Reply:
In doxygen rendering, this looks better.

Currently all of this shows on one line. With this, both the return values will be on separate lines. It is very much a matter of preference.